tests/user_feedback.cpp

```diff
@@ -5,6 +5,8 @@
 
 #include "tests.h"
 
+#include <algorithm>
```

Comment:
Here I don't know where you would put that include.

Reply:
That's the perfect place for it. Just wrap it in warning-disabling bookends:

```suggestion
TOML_DISABLE_WARNINGS
#include <algorithm>
TOML_ENABLE_WARNINGS
```

Reply:
This test code probably doesn't really belong in this file (see my other note), so once it has a new home just wrap it in warning-disabling bookends:

```suggestion
TOML_DISABLE_WARNINGS
#include <algorithm>
TOML_ENABLE_WARNINGS
```